frontend/src/utilities/NavData.tsx

```diff
@@ -53,6 +53,14 @@ const getSettingsNav = (
     });
   }
 
+  if (featureFlagEnabled(dashboardConfig.spec.dashboardConfig.disableAccelerators)) {
+    settingsNavs.push({
+      id: 'accelerator-profile',
+      label: 'Accelerator profiles',
```

Comment:
In the mocks I see "Accelerators" in place of "Accelerator Profiles" and also, I don't see the "New" label. Let me know if I missed any conversations.

Reply:
We have decided to skip "New" label for now. cc @Gkrumbach07

Reply:
@kaedward Can you pls confirm if we are going with "Accelerators" or "Accelerator Profiles" in Sidebar?

Reply:
Accelerators is fine for now. @kaedward Let use know if this should change